### PR TITLE
Add feature to sort groups using shortcut keys

### LINKED ISSUE #6720: Add shortcuts to sort groups up/down/in/out
There is nothing more frustrating to use a great application and discover there has not been put any effort in adding hotkeys for vital function. One of these vital functions would be the up-/down-sorting of Groups. KeePass had this functionality, i don't know why this has been removed.

### PR DESCRIPTION
**Pull request by pruthig**
The code changes in this pull requests pertain to issue # 6720 ( Add shortcut keys to sort groups in ascending/descending order )
To realize this functionality, I've added shortcut keys - [Ctrl + Key_UP_ARROW] to sort the groups in descending order and  [Ctrl + Key_DOWN_ARROW] to sort in ascending order.

Fixes #6720

## Screenshots
When [Ctrl + Key_UP_ARROW] is pressed

image

When [Ctrl + Key_DOWN_ARROW] is pressed

image

## Testing strategy
I manually tested the changes on my Windows 10 machine.

## Type of change
- ✅ New feature (change that adds functionality)

### CODE REVIEW

File: src/gui/group/GroupView.cpp [hunk 1]
```diff
@@ -45,6 +45,13 @@ GroupView::GroupView(Database* db, QWidget* parent)
 
     new QShortcut(Qt::CTRL + Qt::Key_F10, this, SLOT(contextMenuShortcutPressed()), nullptr, Qt::WidgetShortcut);
 
+    // keyboard shortcuts to sort children of a group
+    auto shortcut = new QShortcut(Qt::CTRL + Qt::Key_Down, this);
+    connect(shortcut, &QShortcut::activated, this, [this]() { sortGroupRequested(false); });
```
Comment (droidmonkey) [inline src/gui/group/GroupView.cpp]: Why a new function, just call sortGroups directly right here.

**Review by droidmonkey — CHANGES_REQUESTED** [review]

File: src/gui/group/GroupView.cpp (hunk 1 above)
Comment (pruthig) [reply src/gui/group/GroupView.cpp]: Oh..yes.. Correct.
Initially, I made changes as per SIGNAL - SLOT mechanism and with different strategy. I just removed that function now and it's working fine.
Thanks.

**Review by droidmonkey — APPROVED** [review]

**Comment by pruthig** [comment]
@droidmonkey Hi Jonathan,
Is there some issue with Windows 10? Merge is blocked because of this.

**Comment by phoerious** [comment]
Windows CI needs some attention, yes.

**Comment by droidmonkey** [comment]
The unfortunate thing with this change is we can't show the keyboard shortcut on the context menu because then it would make the shortcut an application wide one since that context menu is defined at the MainWindow. I am merging this for now but I think we need to look into a smarter way to handle our context menus.

**Pull request merged**